lld/test/MachO/objc-category-merging-erase-objc-name-test.s

```diff
@@ -0,0 +1,306 @@
+# REQUIRES: aarch64
```

Comment:
NIT: Let's use a consistent comment character in this file. `;` (rather than `#`) seems to render as a comment here, so that might be best.